deals/types.go

```diff
@@ -23,8 +24,7 @@ type StoreResult struct {
 // StorageDealInfo contains information about a proposed storage deal.
 type StorageDealInfo struct {
 	ProposalCid cid.Cid
-	StateID     uint64
-	StateName   string
+	Status      storagemarket.StorageDealStatus
```

Comment:
This looks like a breaking change.
If this struct was saved in some place in a Datastore, looks like might be a problem. 
Is that the case?

Reply:
Looks like it is stored in the deal store for the storage record stuff I did a while back. Maybe for now I'll stick with the previous `StateID` and `StateName` so we don't break those records. Long term, we should create our own state enum that we map to/from the `storagemarket` one (or any other lotus client of the future)